packages/twenty-front/src/modules/object-metadata/hooks/useApolloMetadataClient.ts

```diff
@@ -1,9 +1,15 @@
 import { useContext } from 'react';
+import { useApolloClient } from '@apollo/client';
 
 import { ApolloMetadataClientContext } from '../context/ApolloClientMetadataContext';
 
 export const useApolloMetadataClient = () => {
   const apolloMetadataClient = useContext(ApolloMetadataClientContext);
+  const apolloCLient = useApolloClient();
```

Comment:
The L is in uppercase!
```suggestion
  const apolloClient = useApolloClient();
```